eslint/keep-i18n-patch-in-sync.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable bsky-internal/keep-i18n-patch-in-sync */
+const LOCALE_DATA_FOLDER = '@formatjs/intl-pluralrules/locale-data/'
+const GEN_MODULE_PATH =
+  '@formatjs/intl-pluralrules/supported-locales.generated.js'
+
+exports.create = function create(context) {
+  delete require.cache[require.resolve(GEN_MODULE_PATH)]
+  const {supportedLocales} = require(GEN_MODULE_PATH)
+  return {
+    Literal(node) {
+      if (typeof node.value !== 'string') {
+        return
+      }
+      if (!node.value.startsWith(LOCALE_DATA_FOLDER)) {
+        return
+      }
+      const code = node.value.slice(LOCALE_DATA_FOLDER.length)
+      if (!supportedLocales.includes(code)) {
```

Comment:
This will catch adding new lines like this:

https://github.com/bluesky-social/social-app/blob/077da0830924d4c040ef095d8349f13fdfdf4372/src/locale/i18n.ts#L81

without also amending the patch below.